lib/connection.ex

```diff
@@ -546,11 +546,11 @@ defmodule Janus.Connection do
            "transaction" => transaction,
            "session_id" => _session_id,
            "handle_id" => _handle_id,
-           "info" => info
-         },
+           "info" => _info
+         } = msg,
          state
        ) do
-    handle_successful_payload(transaction, info, state)
+    handle_successful_payload(transaction, msg, state)
```

Comment:
If such error wasn't caught with tests, something is wrong with the tests.. Also: https://github.com/software-mansion-labs/elixir-janus/pull/9#pullrequestreview-480747923

Reply:
It was, at some point when I changed something I forgot to run it once again.